### ISSUE TITLE
Add Motorcycle, Icon that represents any "New", "Used" category/item... etc

### ISSUE THREAD

**Issue by c-klayaani** [primefaces/primeicons#913]

**Comment by jepsar** [comment]
I think your request is way to specific for a generic icon library.

**Comment by jepsar** [comment]
Yes, I still think your request is very specific. Apart from that, I have no idea what the difference between a used and new.. eh.. basically anything would be visually.

**Comment by jepsar** [comment]
That would be a duplicate of #281

**Comment by onursenture** [comment]
We may consider to add motorcycle. Thanks.

**Comment by c-klayaani** [comment]
Sweet! thanks @onursenture
